lib/earmark.ex

```diff
@@ -318,7 +318,8 @@ defmodule Earmark do
 
   defp _as_html(lines, options) do
     start_link()
-    {blocks, context} = parse(lines, options)
+    ast = {blocks, context} = parse(lines, options)
```

Comment:
I think for the time being we should leave the working code alone.

Reply:
Ah this was just me testing. I've got a bunch of `IO.inspect` lines throughout the code testing things out, this was just one of them :)